languages/elixir/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,43 @@
+Elixir is a dynamically-typed language, meaning that the type of a variable is only checked at run-time. Using the match `=` operator, we can bind a value to a variable name:
+
+```elixir
+variable = 10 # Bind the integer value 10
+```
+
+Re-binding a variable's value is also done performed with the `=` operator. Once defined, a variable's type can change when re-bound.
+
+```elixir
+count = 1 # Bind an integer value of 1
+count = 2 # Re-bind to the new value of 2
+
+# Compiler does not error when binding a new type to the variable
+count = false
+```
+
+Elixir is an [functional-programming language][functional-programming] and requires all functions to be defined in a _module_. The `defmodule` keyword is used to define a module. All modules are available to all other modules at run-time and do not require an _access modifier_ to make them visible to other parts of the program.  A _module_ is analogous to a _class_ in other programming languages.
+
+```elixir
+defmodule Calculator do
+  # ...
+end
+```
+
+_Functions_ can be defined inside of modules. Each function can have zero or more parameters. All parameters are dynamically typed, and the return type is not explicitly declared, it is the type of the value returned. An _access modifier_ can be specified for functions, making only desired functions available for use external to the module. In a function, the value of the last line is _implicitly returned_ to the calling function.
```

Comment:
The previous paragraph states:

> [...] requires all functions to be defined in a _module_

Here it states that they _can_ be defined inside of modules. Which is correct?

Reply:
*named functions, will specify in the doc

Reply:
Same change should be made to the `after.md` document :)

Reply:
done!